src/accel/TrackingManagerOffload.cc

```diff
@@ -0,0 +1,141 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2023 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file accel/TrackingManagerOffload.cc
+//---------------------------------------------------------------------------//
+#include "TrackingManagerOffload.hh"
+
+#include <G4ProcessManager.hh>
+#include <G4ProcessVector.hh>
+#include <G4Track.hh>
+
+#include "corecel/Assert.hh"
+
+#include "ExceptionConverter.hh"
+#include "LocalTransporter.hh"
+#include "SharedParams.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Construct a tracking manager with data needed to offload to Celeritas
```

Comment:
For the "autobrief" doxygen to work, the first line here (and in all the rest of the docs) need a trailing full stop `.` .